### DISCUSSION TITLE
Outstanding SA/SD in terns of QoE

### DISCUSSION THREAD

**Discussion by erato41** [RafaelGB/obsidian-db-folder#109] [General]
Well done, Rafael.🖖🏻
It's brilliant and much better than those acceptable approaches when compared with other Plugins.
I managed to use it to aggregate current notes instantly. And I have also added more columns successfully. And I'm wondering if I can add some more attributes and backlinks within the new columns created as a follow-up action or get better visibility?
I hope you are going well as always and making tremendous progress on this project. Can't wait to get some new updates soon.

Cheers,
Erato

**Comment by RafaelGB** [comment]
Hi @erato41 Thanks for your comment!
Any idea is really welcome in the [Issues](https://github.com/RafaelGB/obsidian-db-folder/issues)  panel. Our next version will bring new visual configurations  and tags column option =)
